js/code.js

```diff
@@ -1,65 +1,52 @@
-var urlBase = 'http://contactical.xyz/LAMPAPI';
-var extension = 'php';
+var urlBase = 'LAMPAPI/';
+var extension = '.php';
 
 var userId = 0;
 var firstName = "";
 var lastName = "";
 
 function doLogin()
 {
-	var userId = 0;
-	var firstName = "";
-	var lastName = "";
-	
+	// Reset loginStatus
+	document.getElementById("loginStatus").innerHTML = "";
+
+	// Retrieve login and password
 	var login = document.getElementById("user").value;
 	var password = document.getElementById("pass").value;
-//	var hash = md5( password );
-	
-  // -- ZACH --
-	// Added because right now if you click the login button without entering anything it still runs the doLogin() function.
-	if (login.length == 0 || password.length == 0)
-	{
-		var badValue = (login.length == 0 ? "Username" : "Password");
-		alert('Please enter your ' + badValue);
-		return false;
-	}
-  
-	document.getElementById("loginStatus").innerHTML = "";
 
-//	var jsonPayload = '{"login" : "' + login + '", "password" : "' + hash + '"}';
-	var jsonPayload = '{"login" : "' + login + '", "password" : "' + password + '"}';
-	var url = urlBase + '/Login.' + extension;
+    // Check if either login or password is blank
+    if (login.length == 0 || password.length == 0) {
+        alert("Please provide a " + (login.length == 0 ? "username" : "password"));
+        return false;
+    }
 
+	// Hash the password
+	var hash = md5(password);
+	
+	// Create jsonPayload and api endpoint
+	var jsonPayload = '{"Login" : "' + login + '", "Password" : "' + hash + '"}';
```

Comment:
```suggestion
	var jsonPayoload = {
	     "Login" : login,
	     "Password" : hash
	};
```

JSON in javascript can be represented like shown above, it doesn't have to be a string.

Reply:
This is awesome! Will add it to the commit :)